packages/frontend-shared/cypress/e2e/e2ePluginSetup.ts

```diff
@@ -191,7 +191,7 @@ async function makeE2ETasks () {
                   query,
                   result,
                   callCount: operationCount[operationName ?? 'unknown'],
-                })
+                }, testState)
```

Comment:
I added testState to the remoteGraphQLIntercept callback so that I could assert that [an operation occurred](https://github.com/cypress-io/cypress/pull/20764/files#diff-18d3d39b03b089d6c1a5c08ea987b6ae379d4dc05307619b14a18fc7f02ef8d3R237). This might be controversial? Suggestions welcome for an alternative approach.

Reply:
This seems similar to `operationCount`, which was merged in yesterday - https://github.com/cypress-io/cypress/pull/20598/files#r831748428. Though operationCount is desgined for a different use, it seems like it could be used here.

I'd have been inclined towards your implementation, but that one landed first. We should probably settle on just one of them :woman_shrugging:

Reply:
I had a similar thought when I initially merged the conflicts, but I settled on leaving both as they don't conflict with each negatively. The operation count could be reimplemented using testState on a per-test basis, but it seems fair to track that internally and offer it first-class for the tests that need it. The operationCount doesn't give me the ability to assert that an operation was called, which was what I was wanting for these tests.

Reply:
operationCount is used precisely once (the second usage is in error, and I'm removing it in an upcoming PR) - I think implementing it per-test isn't a terrible thing.  But with that said, looks good and it doesn't bother me enough to block merging and getting on with our days. :)